azure_sdk_storage_table/src/table/mod.rs

```diff
@@ -96,6 +98,66 @@ impl TableService {
         })
     }
 
+    fn query_entity_collection<T: DeserializeOwned>(
+        &self,
+        table_name: &str,
+        query: Option<&str>,
+        continuation: Option<&Continuation>,
+    ) -> impl Future<Item = EntityCollection<T>, Error = AzureError> {
+        let mut path = table_name.to_owned();
+        path.push_str("?");
+        if let Some(clause) = query {
+            path.push_str(clause);
+        }
+        if let Some(cont) = continuation {
+            path.push_str("&NextPartitionKey=");
+            path.push_str(&cont.next_partition_key);
+            path.push_str("&NextRowKey=");
+            path.push_str(&cont.next_row_key);
+        }
+
+        let req = self.request_with_default_header(path.as_str(), &Method::GET, None);
+
+        done(req)
+            .from_err()
+            .and_then(move |future_response| check_status_extract_headers_and_body(future_response, StatusCode::OK))
+            .and_then(move |(headers, body)| done({
+                serde_json::from_slice::<EntityCollection<T>>(&body)
+                .map(|ec| {
+                    ec.continuation = None;
```

Comment:
TODO here. Need to extract headers `x-ms-continuation-NextPartitionKey` and `x-ms-continuation-NextRowKey` if set. https://docs.microsoft.com/en-us/rest/api/storageservices/query-timeout-and-pagination